### PR TITLE
feat(datasets) Update FDS list of supported datasets

### PR DESCRIPTION
**Pull request by adam-narozniak**
## Issue
Some of the datasets on HugggingFace changed the signature and are not discoverable under the raw e.g. `mnist` name but rather `ylecun/mnist`.

### Description

The original references still work, but if someone is searching for a dataset, now he finds a different name.

## Proposal
Extend the list of supported datasets by the datasets that we have already tested for but are now available under different names.

### CODE REVIEW

**Review by jafermarq — APPROVED** [review]

File: datasets/flwr_datasets/utils.py
Comment (jafermarq) [inline datasets/flwr_datasets/utils.py:45]:
```suggestion
    "flwrlabs/femnist",
    "flwrlabs/ambient-acoustic-context",
    "flwrlabs/ucf101",
    "flwrlabs/shakespeare",
    "flwrlabs/celeba",
```

Comment (jafermarq) [reply datasets/flwr_datasets/utils.py:45]: @adam-narozniak should we add the new datasets under `flwrlabs/`?

Comment (jafermarq) [reply datasets/flwr_datasets/utils.py:45]: nevermind. let's wait until the other PRs are merged. still i think a couple of dataset will need to be added

Comment (adam-narozniak) [reply datasets/flwr_datasets/utils.py:44]: Thanks for pointing that out, yes a few datasets are still lagging with the tests. They should be added in new PRs. The purpose of this PR is to just handle different namespace of the datasets that already have tests but can be found under different names

**Review by jafermarq — APPROVED** [review]

**Pull request merged**
